server/channels/web/oauth_test.go

```diff
@@ -678,7 +679,10 @@ func HTTPGet(url string, httpClient *http.Client, authToken string, followRedire
 
 func closeBody(r *http.Response) {
 	if r != nil && r.Body != nil {
-		io.ReadAll(r.Body)
+		bodyBytes ,err := io.ReadAll(r.Body)
+		if err != nil {
+			return 
+		}
 		r.Body.Close()
 	}
 }
```

Comment:
There is already a `closeBody` function in the `model` package. Can we use that one instead?

Reply:
Apologies, I couldn't find the `closeBody` function in the `model` package.

Reply:
https://github.com/mattermost/mattermost/blob/9fee53fb1470973445a10a79f4c4cdd5460b0325/server/public/model/client4.go#L115-L120

Does that help?

Reply:
@hanzei Can you check it now?